### PR TITLE
Remove Member Count Feature Changes

### PR DESCRIPTION
**Pull request by Ninad-S**
Pull request to merge remove-member-count changes to develop brach

### CODE REVIEW

File: src/components/DirectoryOrgs.tsx [hunk 1]
```diff
@@ -33,9 +33,7 @@ const OrgDirectoryCards: FC<Props> = async ({ club }) => {
             <div className="h-7 w-7 text-white">
               <GroupIcon />
             </div>
-            <div className="ml-1 h-fit w-fit text-xs font-bold text-white">
-              {30} Members
-            </div>
+            
           </div>
```
Comment (RubenOlano) [inline src/components/DirectoryOrgs.tsx:37]: For this, we're good to also remove the group icon

**Review by RubenOlano — CHANGES_REQUESTED** [review]
Good small change but just need that one change as well

File: src/components/DirectoryOrgs.tsx (hunk 1 above)
Comment (Ninad-S) [reply src/components/DirectoryOrgs.tsx]: Yes I removed it and checked in the code

**Review by Ninad-S — COMMENTED** [review]
Removed Group Icon

**Review by RubenOlano — APPROVED** [review]
Good Job!

**Pull request merged**